app/services/users.js

```diff
@@ -26,3 +28,94 @@ exports.existUserMail = async mail => {
     return new Error(`signUpUser Error => ${error.message}`);
   }
 };
+
+exports.getUserByEmailAndPassword = async (mail, newPassword) => {
```

Comment:
I think that service is a function that it to do a unique operation. It the name of service is `getUserByEmailAndPassword ` I would wait something  like this `userModel.findOne({ where: { mail } }).catch(dataBaseError})` and the other actions I would create them in a interactor. What do you think ?